conf/local-config.json

```diff
@@ -28,5 +28,8 @@
   "salt_snapshot_activates_in_seconds": 180,
   "salt_snapshot_expires_after_seconds": 180000,
   "is_auth_disabled": true,
-  "filter_key_over_cut_off_days": false
+  "oauth2_callback_url": "http://localhost:8089/oauth2-callback",
+  "github_client_key": "generate client key",
```

Comment:
Have we set those value in `k8s-deployment`?

Reply:
Yep, but I just added it here in case anyone wanted to test OAuth 2.0 themselves (had to test it as part of the upgrade). The fields won't be used as long as `is_auth_disabled` is set to `true`, and I've updated the README to explain how to test OAuth 2.0 locally